docs/add-ins/delegate-access.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Enable an Outlook add-in to support delegate access scenarios
+description: Briefly describes delegate access and discusses how to configure add-in support.
+ms.topic: article
+ms.date: 02/11/2019
+localization_priority: Normal
+---
+
+# How to enable delegate access scenarios in an Outlook add-in (preview)
+
+There are a few steps you should follow to configure your Outlook add-in to support delegate access. These include enabling the `SupportsSharedFolders` property in the manifest and using the `SharedProperties` object.
+
+## What is delegate access?
+
+Consider a business setting where an executive enables their assistant to manage the executive's mail and calendar. Depending on the level of access, the assistant or delegate can respond on the mailbox owner's behalf and create new appointments, among other tasks. To learn more about delegate access and how to give delegates permission, see [Allow someone else to manage your mail and calendar](https://support.office.com/article/allow-someone-else-to-manage-your-mail-and-calendar-41c40c04-3bd1-4d22-963a-28eafec25926).
+
+## Supported permissions
+
+You should know which delegate permissions the JavaScript for Office API supports.
+
+**Table 1 Supported permissions for delegate access**
```

Comment:
```suggestion
```
You don't really need the table label if you precede it with the paragraph describing the table.